js_modules/dagit/packages/core/src/runs/RunRoot.tsx

```diff
@@ -105,11 +109,13 @@ const RunById: React.FC<{data: RunRootQuery | undefined; runId: string}> = (prop
 
   if (data.pipelineRunOrError.__typename !== 'PipelineRun') {
     return (
-      <NonIdealState
-        icon="error"
-        title="No Run"
-        description={'The run with this ID does not exist or has been cleaned up.'}
-      />
+      <Box padding={{vertical: 64}}>
```

Comment:
Ahh i noticed this in another spot on my branch too, I think we may need to put the padding inside the ideal state to match the old blueprint one...